packages/overlay/src/OverlayTrigger.ts

```diff
@@ -76,11 +86,17 @@ export class OverlayTrigger extends LitElement {
     @property({ type: Boolean, reflect: true })
     public disabled = false;
 
+    @property({ type: Boolean, attribute: false })
+    public hasLongpressContent = false;
+
+    private longpressDescriptor?: HTMLElement;
     private clickContent?: HTMLElement;
     private longpressContent?: HTMLElement;
     private hoverContent?: HTMLElement;
     private targetContent?: HTMLElement;
 
+    public _longpressId = `longpress-describedby-descriptor`;
```

Comment:
The `_*` syntax _implies_ that this is a "private" property, we should do so with TS as well `private _longpressId`...